package.json

```diff
@@ -77,11 +78,13 @@
         "@angular/platform-browser": "17.3.1",
         "@angular/platform-browser-dynamic": "17.3.1",
         "@angular/router": "17.3.1",
+        "@changesets/cli": "^2.27.1",
         "@rollup/plugin-commonjs": "25.0.7",
         "@rollup/plugin-json": "6.1.0",
         "@rollup/plugin-node-resolve": "15.2.3",
         "@rollup/plugin-terser": "0.4.4",
         "@rollup/plugin-typescript": "11.1.6",
+        "@svitejs/changesets-changelog-github-compact": "^1.1.0",
```

Comment:
Just curious why this didn't save the exact version when `save-exact` is specified in `npmrc`?